Description: Fixes the compatibility with the version of the maven-jar-plugin in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/pom.xml
+++ b/pom.xml
@@ -175,10 +175,12 @@
                 <artifactId>maven-jar-plugin</artifactId>
                 <version>2.4</version>
                 <configuration>
-                    <useDefaultManifestFile>true</useDefaultManifestFile>
                     <excludes>
                         <exclude>**/*.java</exclude>
                     </excludes>
+                    <archive>
+                        <manifestFile>target/classes/META-INF/MANIFEST.MF</manifestFile>
+                    </archive>
                 </configuration>
             </plugin>
             <plugin>
